internal/graphql/graph/schema.resolvers.go

```diff
@@ -49,31 +50,53 @@ func (r *queryResolver) Executors(ctx context.Context) ([]*testkube.Executor, er
 	panic(fmt.Errorf("not implemented: Executors - executors"))
 }
 
+func getExecutors(client *executors.ExecutorsClient) ([]testkube.ExecutorDetails, error) {
+	execs, err := client.List("")
+	if err != nil {
+		return nil, err
+	}
+
+	execsDetails := []testkube.ExecutorDetails{}
+	for _, item := range execs.Items {
+		execsDetails = append(execsDetails, executorsmapper.MapExecutorCRDToExecutorDetails(item))
+	}
+	return execsDetails, nil
+}
+
 // Executors is the resolver for the executors field.
 func (r *subscriptionResolver) Executors(ctx context.Context) (<-chan []testkube.ExecutorDetails, error) {
 	ch := make(chan []testkube.ExecutorDetails)
 
 	// TODO You can (and probably should) handle your channels in a central place outside of `schema.resolvers.go`.
 	// For this example we'll simply use a Goroutine with a simple loop.
 	go func() {
+
+		execs, err := getExecutors(r.Client)
```

Comment:
I would log error in thiis func

Reply:
In case, something fails in below code

Reply:
thanks, logged 👍 although I think that it will need to be rather solved globally